### ISSUE TITLE
Network in nat mode keeps recreating

### ISSUE THREAD

**Issue by rsokolkov** [dmacvicar/terraform-provider-libvirt#99]
This behavior only for nat mode. e.g. "none" mode works fine.

```
tf|⇒ terraform apply
libvirt_network.adm_net: Refreshing state... (ID: ec1b7cd1-4056-495f-bc57-71f87eb112be)
libvirt_network.adm_net: Destroying...
libvirt_network.adm_net: Destruction complete
libvirt_network.adm_net: Creating...
  addresses.#: "" => "1"
  addresses.0: "" => "10.30.0.1/24"
  bridge:      "" => "<computed>"
  domain:      "" => "lab"
  mode:        "" => "nat"
  name:        "" => "rs-terraform-test"
  running:     "" => "true"
libvirt_network.adm_net: Still creating... (4s elapsed)
libvirt_network.adm_net: Creation complete

Apply complete! Resources: 1 added, 0 changed, 1 destroyed.
```

Terraform file
```
provider "libvirt" {
  uri = "qemu+ssh://root@pdt2/system"
}

resource "libvirt_network"
"adm_net" {
  name = "rs-terraform-test"
  domain = "lab"
  mode = "nat"
  addresses = ["10.30.0.1/24"]
}
```

Plugin built from aeaee868666a6017b9bd0539c618711c4d15ea0d
Terraform v0.8.6
Libvirt version 1.3.1-1ubuntu10.7

**Comment by rsokolkov** [comment]
I should use ["10.30.0.0/24"] instead of ["10.30.0.1/24"]. Updated docs, because it was not clear

**Comment by inercia** [comment]
I guess this was closed by #101, but please reopen otherwise...
